tests/test_loss.py

```diff
@@ -0,0 +1,31 @@
+import inspect
+import unittest
+
+import monai
+import torch
+from parameterized import parameterized
+from test_utils import TestCase, func_req_additional_args
+
+from trainlib.loss import get_loss
+
+monai_losses = [item[0] for item in inspect.getmembers(monai.losses) if item[0].endswith("Loss")]
+torch_losses = [item[0] for item in inspect.getmembers(torch.nn) if item[0].endswith("Loss")]
+monai_losses = [x for x in monai_losses if not func_req_additional_args(getattr(monai.losses, x))]
+torch_losses = [x for x in torch_losses if not func_req_additional_args(getattr(torch.nn, x))]
+
+LOSSES = monai_losses + torch_losses
+
+
+class TestLoss(TestCase):
+    @parameterized.expand(LOSSES)
+    def test_init(self, loss_name):
+        "Simple test if losses can be initialized"
+        config = self.config
+        config.loss = {loss_name: {}}
```

Comment:
Does this change a class variable?

Reply:
The default config.loss is: 
```python
{'DiceCELoss': {'include_background': False, 'softmax': True, 'to_onehot_y': True}}
```
For testing, it is changed to the respective loss to test. E.g, if `DiceLoss` is tested, config.loss would be changed to: 

```python
{'DiceLoss': {}}
```
As I only pass loss functions that have default args during init, the empty dictionary is fine.